### PR TITLE
Fix color picker opening partly off screen when client is maximized

### PR DESCRIPTION
**Pull request by Keirathi**
Not a huge deal, but fixes the color picker dialogs opened by screen marker and chat color configs and keeps them from opening partially off screen.  Also opens them in slightly more logical place (centered on the button you clicked, instead of at the top of the window) all the time.

### CODE REVIEW

**Comment by Keirathi** [comment]
Is that a problem?

Hmm, I'm not sure how to get the exact dimensions of a window in Java, but my estimation the window is 400x390 instead of 400x400.

Anyways, don't need the pack() in there to fix left or right off screen, but setPositionRelativeTo() made buttons very low on the screen open slightly off the bottom of the screen.  Not so far down that you couldn't still use all of the components, but it didn't look great.  pack() fixed that issue, but if it caused another then I wasn't able to find it.

**Comment by raiyni** [comment]
Yes

File: runelite-client/src/main/java/net/runelite/client/plugins/screenmarkers/ui/ScreenMarkerPanel.java [hunk 1]
```diff
@@ -27,6 +27,7 @@
 
 import java.awt.BorderLayout;
 import java.awt.Color;
+import java.awt.Component;
```
Comment (raiyni) [inline runelite-client/src/main/java/net/runelite/client/plugins/screenmarkers/ui/ScreenMarkerPanel.java]: unused import

File: runelite-client/src/main/java/net/runelite/client/ui/components/colorpicker/RuneliteColorPicker.java [hunk 2]
```diff
@@ -342,6 +344,15 @@ private void updateHex()
 		updatePanels();
 	}
 
+	public void fitInWindowBounds()
```
Comment (raiyni) [inline runelite-client/src/main/java/net/runelite/client/ui/components/colorpicker/RuneliteColorPicker.java]: this doesn't work on monitors that have a different resolution/scaling than main monitor.

**Comment by raiyni** [comment]
added in https://github.com/runelite/runelite/commit/707ab78cdb6da090032c59f72e4ebfbc245126b1

**Pull request closed without merging**